src/shared-components/forms/style.js

```diff
@@ -0,0 +1,124 @@
+import styled from '@emotion/styled';
+import { css } from '@emotion/core';
+
+import { style as TYPOGRAPHY_STYLE } from '../typography';
+import { COLORS, BOX_SHADOWS, SPACING, ANIMATION } from '../../constants';
+
+export const HelperList = styled.ul`
+  list-style-type: none;
+  margin: 8px 0;
```

Comment:
8px should be one of our spacing constants

Reply:
fixed